container_pipeline/workers/base.py

```diff
@@ -31,7 +31,7 @@ def notify(self, data):
 
     def export_logs(self, logs, destination):
         """"Write logs in given destination"""
-        self.logger.debug('Writing build logs to NFS share')
+        self.logger.info('Writing build logs to NFS share')
```

Comment:
Changed this to `info` as this seems more like an informative message we're storing in the logs.